src/components/Table/Table.html

```diff
@@ -0,0 +1,74 @@
+<table class="table">
+  <thead class="table__head">
+    <tr>
+      <th class="table__th"></th>
+      <th class="table__th table__th_number">
+        <button class="mdc-button table__th-button">
+          No.
+          <svg data="number" aria-hidden="true" class="mdc-button__icon table__th-icon table__th-icon_number" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
```

Comment:
please create a folder called `assets/images` and move this svg there. BTW - maybe there is such arrow in material icons?